frontend/common/src/components/form/MultiSelect/MultiSelectFieldV2.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import { uniqueId } from "lodash";
```

Comment:
I know this is just the dev dependency but prefer importing the function from the single file for smaller bundle.  https://stackoverflow.com/a/35251059

Reply:
Ah thanks! total agreement -- I've def seen the bundlesize diff it makes. Copied this and missed it :)

Reply:
```suggestion
import uniqueId from "lodash/uniqueId";
```